<ui version="4.0" >
 <author></author>
 <comment>This file is part of the xTuple ERP: PostBooks Edition, a free and
open source Enterprise Resource Planning software suite,
Copyright (c) 1999-2014 by OpenMFG LLC, d/b/a xTuple.
It is licensed to you under the Common Public Attribution License
version 1.0, the full text of which (including xTuple-specific Exhibits)
is available at www.xtuple.com/CPAL.  By using this software, you agree
to be bound by its terms.</comment>
 <class>dspInvoiceInformation</class>
 <widget class="QWidget" name="dspInvoiceInformation" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Invoice Information</string>
  </property>
   <layout class="QVBoxLayout" >
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>5</number>
      </property>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>5</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>7</number>
          </property>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>5</number>
            </property>
            <item>
             <widget class="QLabel" name="_invoiceNumberLit" >
              <property name="text" >
               <string>&amp;Invoice #:</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="buddy" >
               <cstring>_invoiceNumber</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="InvoiceLineEdit" name="_invoiceNumber" />
            </item>
            <item>
             <widget class="QPushButton" name="_invoiceList" >
              <property name="text" >
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>5</number>
            </property>
            <item>
             <widget class="QLabel" name="_custPoNumberLit" >
              <property name="text" >
               <string>Customer P/O #:</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="_custPoNumber" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>7</hsizetype>
                <vsizetype>5</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="alignment" >
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>7</number>
          </property>
          <item>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>0</number>
            </property>
            <item>
             <widget class="CustCluster" name="_cust" >
              <property name="focusPolicy" >
               <enum>Qt::StrongFocus</enum>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Preferred</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>5</number>
            </property>
            <item>
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>5</number>
              </property>
              <item>
               <layout class="QVBoxLayout" >
                <property name="margin" >
                 <number>0</number>
                </property>
                <property name="spacing" >
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="_invoiceDateLit" >
                  <property name="text" >
                   <string>Invoice Date:</string>
                  </property>
                  <property name="alignment" >
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="_shipDateLit" >
                  <property name="text" >
                   <string>Ship Date:</string>
                  </property>
                  <property name="alignment" >
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" >
                <property name="margin" >
                 <number>0</number>
                </property>
                <property name="spacing" >
                 <number>0</number>
                </property>
                <item>
                 <layout class="QVBoxLayout" >
                  <property name="margin" >
                   <number>0</number>
                  </property>
                  <property name="spacing" >
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="DLineEdit" name="_invoiceDate" >
                    <property name="enabled" >
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="DLineEdit" name="_shipDate" >
                    <property name="enabled" >
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer>
                  <property name="orientation" >
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType" >
                   <enum>QSizePolicy::Expanding</enum>
                  </property>
                  <property name="sizeHint" >
                   <size>
                    <width>11</width>
                    <height>10</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>5</number>
              </property>
              <item>
               <widget class="QLabel" name="_invoiceAmountLit" >
                <property name="text" >
                 <string>Invoice Amount:</string>
                </property>
                <property name="alignment" >
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="XLabel" name="_invoiceAmount" >
                <property name="minimumSize" >
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text" >
                 <string/>
                </property>
                <property name="alignment" >
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>0</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>98</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>5</number>
            </property>
            <item>
             <widget class="QPushButton" name="_close" >
              <property name="text" >
               <string>&amp;Close</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="_view" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="text" >
               <string>&amp;View Details</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="_print" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="text" >
               <string>&amp;Print</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>7</number>
      </property>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="_billtoLit" >
          <property name="text" >
           <string>Bill-to:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="XLineEdit" name="_billToName" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="XLineEdit" name="_billToAddress1" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="XLineEdit" name="_billToAddress2" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="XLineEdit" name="_billToAddress3" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>5</number>
          </property>
          <item>
           <widget class="XLineEdit" name="_billToCity" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="XLineEdit" name="_billToState" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="maximumSize" >
             <size>
              <width>40</width>
              <height>32767</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="XLineEdit" name="_billToZip" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="XLineEdit" name="_billToPhone" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="_shiptoLit" >
          <property name="text" >
           <string>Ship-to:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="XLineEdit" name="_shipToName" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="XLineEdit" name="_shipToAddress1" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="XLineEdit" name="_shipToAddress2" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="alignment" >
           <set>Qt::AlignLeading</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="XLineEdit" name="_shipToAddress3" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>5</number>
          </property>
          <item>
           <widget class="XLineEdit" name="_shipToCity" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="XLineEdit" name="_shipToState" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="maximumSize" >
             <size>
              <width>40</width>
              <height>32767</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="XLineEdit" name="_shipToZip" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="XLineEdit" name="_shipToPhone" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="_notesLit" >
        <property name="text" >
         <string>Notes:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="XTextEdit" name="_notes" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="_applicationsLit" >
      <property name="text" >
       <string>Applications:</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="XTreeWidget" name="_arapply" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>7</hsizetype>
        <vsizetype>7</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>XLineEdit</class>
   <extends>QLineEdit</extends>
   <header>xlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>DLineEdit</class>
   <extends>XLineEdit</extends>
   <header>datecluster.h</header>
  </customwidget>
  <customwidget>
   <class>CustCluster</class>
   <extends>QWidget</extends>
   <header>custcluster.h</header>
  </customwidget>
  <customwidget>
   <class>InvoiceLineEdit</class>
   <extends>XLineEdit</extends>
   <header>invoicelineedit.h</header>
  </customwidget>
  <customwidget>
   <class>XTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>xtreewidget.h</header>
  </customwidget>
  <customwidget>
   <class>XTextEdit</class>
   <extends>QTextEdit</extends>
   <header>xtextedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_invoiceNumber</tabstop>
  <tabstop>_invoiceList</tabstop>
  <tabstop>_cust</tabstop>
  <tabstop>_invoiceDate</tabstop>
  <tabstop>_shipDate</tabstop>
  <tabstop>_billToName</tabstop>
  <tabstop>_billToAddress1</tabstop>
  <tabstop>_billToAddress2</tabstop>
  <tabstop>_billToAddress3</tabstop>
  <tabstop>_billToCity</tabstop>
  <tabstop>_billToState</tabstop>
  <tabstop>_billToZip</tabstop>
  <tabstop>_billToPhone</tabstop>
  <tabstop>_shipToName</tabstop>
  <tabstop>_shipToAddress1</tabstop>
  <tabstop>_shipToAddress2</tabstop>
  <tabstop>_shipToAddress3</tabstop>
  <tabstop>_shipToCity</tabstop>
  <tabstop>_shipToState</tabstop>
  <tabstop>_shipToZip</tabstop>
  <tabstop>_shipToPhone</tabstop>
  <tabstop>_notes</tabstop>
  <tabstop>_arapply</tabstop>
  <tabstop>_view</tabstop>
  <tabstop>_print</tabstop>
  <tabstop>_close</tabstop>
 </tabstops>
 <includes/>
 <resources/>
 <connections>
  <connection>
   <sender>_close</sender>
   <signal>clicked()</signal>
   <receiver>dspInvoiceInformation</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>602</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
